docs/extensions/reference/_posts/1970-01-06-CLI.md

```diff
@@ -327,17 +327,18 @@ By default, `shoutem pack` command will do `npm run build` in both `app` and `se
 
 Run this command from the extension folder.
 
-- ##### shoutem configure [--release]
+- ##### shoutem configure [--release] [--production]
 
 Runs the platform's configure script to sync the local app with native changes to local extensions.
 The flag `--release` is used to switch the local app to the last published configuration on the server.
-Call the `shoutem configure` command again without the flag to switch back to the current development configuration.
+The flag `--production` is used to configure the app's bundle ID for iOS and Android to match the one provided in the Builder. It also activates the code push feature, analytics and push notifications.
 
 The command should be called when:
 - new extension is created, installed to the current app & pushed
 - changing native code of any of the extensions from the extensions/ directory
 - changing cocoapods or gradle dependencies of any of the extensions from the extensions/ directory
 - switching between published (`shoutem configure --release`) and development (`shoutem configure`) configuration
```

Comment:
`--release` flag, doesn't use published configuration, `--production` does. `--release` just bundles downloaded configuration, so app doesn't download it on each start. Also, `--production` flag automatically sets `release` flag.